docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/structuredDocument/fields/dictionary/dictionary.rst

```diff
@@ -4,22 +4,64 @@ Dictionary field
 
 .. toctree::
 
----------
+Dictionary is a list of records in a table form, e.g. list of employees (First name, Last name, Age, Address).
+Data from dictionary could be used in envelope.
 
-Dictionary - is a list of records in a systematic form, e.g. list of employee(Name, Surname, Age, Address).
-Data from dictionary could be used in envelope:
+Create Dictionary Field
+=======================
 
-1. Create dictionary field in template.
+1. Navigate to https://staging.whitedoc.space/template/form/editor
 
-2. Set dictionary and column from which recipient will select data to field.
+2. Drag and drop dictionary field to the document.
 
-Dictionary template field
-=========================
-To add dictionary field to template drag and drop dictionary field from the fields bar to the document. Select dictionary, receiver role and column of the dictionary.
-Also it's possible to set "Allow custom value" it allows to enter custom value on envelope stage (when receiver fill in the document).
+3. Next window should appear:
 
+.. image:: pic_dictionary/DictionaryCreationForm.png
+   :width: 600
+   :align: center
 
-Dictionary envelope field
-=========================
-User need to select value from the list of records values. If allow custom value attribute is true - user can enter custom value.
-After value was set to field and processing flow was continued - value of dictionary field is blocked. Any updates in dictionary doesn't apply to this field.
\ No newline at end of file
+4. Fill the "Name" attribute of the field (should be unique per document).
+
+5. Select a role from "Role name" menu which will be filling this field.
+
+6. Fill the "Placeholder" attribute for this field (not mandatory, if you won't fill this field, name will be used as placeholder).
```

Comment:
..., if you won't fill this field, name will be used as placeholder). -> ..., if you don't fill this field, name will be used as placeholder

Reply:
Done